src/main/python/janusgraph_python/core/attribute/GeoPredicate/Geo.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .GeoContains import GeoContains
+from .GeoWithin import GeoWithin
+from ...datatypes.GeoShape import GeoShape
+
+
+class Geo(object):
+    """
+    This class defines all the Geo Predicates.
+    """
+
+    contains = GeoContains()
+    within = GeoWithin()
+
+    @staticmethod
+    def geoContains(value):
+        """ The class is used for JanusGraph geoContains predicate.
+
+        GeoContains predicate holds true when one object, is contained by another. Considering the example in docs,
```

Comment:
Not that important and I'm not a native speaker, but I believe there shouldn't be a comma here: _when one object, is contained by another_
Apart from that: I don't understand the second sentence. Which example from which docs? I think these docstrings should be self-contained and not reference some other docs that the reader doesn't have available when reading this.

Reply:
@FlorianHockmann : Thanks. Nice catch :+1 . Also, I've updated the statements here, and rather made it self explanatory.